srcpkgs/apache/template

```diff
@@ -63,6 +62,10 @@ pre_configure() {
 
 	cat ${FILESDIR}/xbps.layout >> config.layout
 	if [ "$CROSS_BUILD" ]; then
+		# pcre{,2}-config detection doesn't work on cross
+		# since httpd-2.4.53, manually export PCRE_CONFIG
+		# to let configure know where it is
+		export PCRE_CONFIG="${XBPS_CROSS_BASE}/usr/bin/pcre2-config"
```

Comment:
Could you report this issue upstream and watch it so we know when we can drop this workaround?

Reply:
Turns out it has already been reported: 
https://bz.apache.org/bugzilla/show_bug.cgi?id=66000
I'll follow this issue and wait for it to be fixed.

Reply:
in that case, --with-pcre with the full path might be prettier

Reply:
@ologantr mentioned on IRC that that wouldn't work on cross, so let's use $PCRE_CONFIG for now